src/typescript/rollup-plugin-fable/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "rollup-plugin-fable",
+  "version": "0.0.0",
```

Comment:
This is already updated on master. 

As an aside, I usually use something like https://www.npmjs.com/package/np for publishing.

When doing this you get a version bump + tagging + publishing automatically.